savant/utils/artist/artist_gpumat.py

```diff
@@ -52,7 +52,7 @@ def add_text(
         bg_color: Optional[Tuple[int, int, int, int]] = (0, 0, 0, 255),  # black
         padding: Tuple[int, int, int, int] = (0, 0, 0, 0),
         anchor_point_type: Position = Position.CENTER,
-    ) -> int:
+    ) -> Tuple[Tuple[int, int], int]:
```

Comment:
We have a release with non-breaking changes. It's certainly not a critical change, but if someone uses the return value of the add_text method, users will get an error. Is it possible to draw without these changes?

Reply:
@dorgun @abramov-oleg I think it is ok.

Reply:
Centering draw objects (such as boxes) vertically against text characters (and not text entire text box) can be challenging without this change.